src/components/Autocomplete.vue

```diff
@@ -281,7 +288,11 @@ export default {
           throw new Error('Network response was not ok.')
         })
         .then(response => {
-          this.results = this.setResults(response)
+          if(this.setResults){
```

Comment:
and then this logic could move inside the originally named `setResults` function

Reply:
That's a reasonable suggestion. I've added it to my fork.